src/plone/app/multilingual/locales/es/LC_MESSAGES/plone.app.multilingual.po

```diff
@@ -3,169 +3,176 @@ msgid ""
 msgid "Multilingual Support"
-msgstr ""
+msgstr "Soporte Multiidioma"
 
 #: ../configure.zcml:159
 msgid "Multilingual Support [uninstall]"
-msgstr ""
+msgstr "Soporte Multiidioma [desinstalar]"
 
 #: ../browser/translate.py:54
 msgid "Need a field"
-msgstr ""
+msgstr "Se necesita un campo"
 
 #: ../browser/controlpanel.py:174
 msgid "Not translated"
-msgstr ""
+msgstr "No traducidos"
 
 #: ../browser/templates/migration.pt:128
 msgid "Reindex"
-msgstr ""
+msgstr "Refrescar"
 
 #: ../browser/templates/migration.pt:174
 msgid "Relocate"
-msgstr ""
+msgstr "Recolocar"
 
 #: ../browser/migrator.py:127
 msgid "Relocate content to the proper root language folder"
-msgstr ""
+msgstr "Recolocar el contenido en el idioma adecuado"
 
 #: ../browser/controlpanel.py:38
 msgid "Save"
-msgstr ""
+msgstr "Guardar"
 
 #: ../interfaces.py:169
 msgid "Search for closest translation in parent's content chain."
-msgstr ""
+msgstr "Buscar la traducción más cercana entre los ancestros."
 
 #: ../interfaces.py:172
 msgid "Show user dialog with information about the available translations."
-msgstr ""
+msgstr "Mostrar una pantalla con información sobre las traducciones disponibles."
 
 #: ../browser/templates/mmap.pt:55
 msgid "Site Setup"
-msgstr ""
+msgstr "Configuración del Sitio"
 
 #: ../browser/templates/not_translated_yet.pt:23
 msgid "This content is not translated yet to the language requested."
-msgstr ""
+msgstr "El contenido no está traducido al idioma solicitado."
 
 #: ../browser/templates/not_translated_yet.pt:47
 msgid "This item doesn't have any translation yet. You can return to the portal site:"
-msgstr ""
+msgstr "Este elemento no tiene traducciones. Puede volver al sitio web:"
 
 #: ../browser/templates/migration.pt:204
 msgid "Transfer"
-msgstr ""
+msgstr "Transferir"
 
 #: ../browser/migrator.py:50
 msgid "Transfer multilingual catalog information"
-msgstr ""
+msgstr "Transferir información del catálogo multiidioma"
 
 #: ../configure.zcml:159
 msgid "Uninstall, removes multilingual content support with plone.app.multilingual"
-msgstr ""
+msgstr "Desinstalar, elimina el soporte de contenido multiidioma de plone.app.multilingual"
 
 #: ../upgrades.zcml:29
 msgid "Update bundle regristration"
-msgstr ""
+msgstr "Actualizar el registro "
 
 #. Default: "This object is going to be a translation to ${DYNAMIC_CONTENT} from: <ul> <li><a href=\"${DYNAMIC_CONTENT}\" class=\"link-overlay\"> <span>${DYNAMIC_CONTENT}</span> <span>${DYNAMIC_CONTENT}</span></a></li> </ul> <span>x</span>"
 #: ../browser/templates/add-form-is-translation.pt:14
 msgid "add-form-is-translation"
 msgstr ""
+"Este elemento va a ser una traducción al $\n"
+"{DYNAMIC_CONTENT} de:  <ul> <li><a href=\\\"$\n"
+"{DYNAMIC_CONTENT}\\\" class=\\\"link-overlay\\\"> <span>$\n"
+"{DYNAMIC_CONTENT}</span> <span>${DYNAMIC_CONTENT}</\n"
+"span></a></li> </ul> <span>x</span>\""
 
 #. Default: "assets"
 #: ../browser/setup.py:139
 msgid "assets_folder_id"
-msgstr ""
+msgstr "recursos"
```

Comment:
@erral what's your opinion on https://github.com/plone/plone.app.multilingual/commit/0be09c3897cfbb7b4d6c3fa57e5d3febbcbdb72e#commitcomment-23479208

I'd rather not translate the "assets_folder_id" by default